<launch>
  <!-- if using gazebo, launch baxter_gazebo -->
  <!-- use sensor information -->
  <!-- MoveIt planner -->
  <node name='enable_robot' pkg='baxter_tools' type='enable_robot.py' args='-e' />
  <node name="action_server" pkg="baxter_interface" type="joint_trajectory_action_server.py" />
  <include file="$(find lightning)/scripts/experiments/baxter/baxter_grippers.launch"/>
  </launch>
